nats/Makefile

```diff
@@ -8,7 +8,7 @@ NAME          := $(shell cargo metadata --no-deps --format-version 1 | jq -r '.p
 VERSION       := $(shell cargo metadata --no-deps --format-version 1 | jq -r '.packages[].version')
 LIBNAME       := $(shell echo $(NAME) | tr '-' '_')
 VENDOR        ?= wasmcloud
-REVISION	  ?= 0
+REVISION	  ?= 2
```

Comment:
This variable, if you check out the GH action, will be conditionally updated to the # of the run that is taking place. The `?=` will initialize the variable if it's not already defined in the environment already so it is needed. Basically, no harm in updating this, but it will be overwritten in the GH action so you might see a different value when this is published to wasmcloud.azurecr.io

Reply:
Ah!! TIL something new. I can set it back to zero, or, as you say, it doesn't really matter